app/src/main/res/xml/pref_automation.xml

```diff
@@ -3,6 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <PreferenceCategory
+        android:key="@string/automation"
```

Comment:
android:key shouldn't be "@string/key_..." ?

Reply:
It should be key_automation... All keys are defined as untranslatable string resources